### PR TITLE
BD-3497: Add link aliasing example for anchor and capture tag

### PR DESCRIPTION
**Pull request by lydia-xie**
### Why are you making this change? (required)

- Reformat link aliasing examples section 
- Add a new example for anchor and capture tag

### Related PRs, issues, or features (optional)

- Closes [BD-3497](https://jira.braze.com/browse/BD-3497)

### Feature release date (optional)

- N/A

### Contributor checklist

- [ ] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/email/templates/link_aliasing.md [hunk 1]
```diff
@@ -76,17 +76,54 @@ When a Content Block without a `lid` value is inserted into a new message, the l
...
+| https://www.braze.com#bookmark1?utm_source=email | https://www.braze.com?lid=eqslgd5a9m3y#bookmark1?utm_source=email |
+{: .reset-td-br-1 .reset-td-br-2}
+
+### Link with anchor and capture tag
+
+**Logic:** When using link aliasing with URLs that contain anchors (#), Braze expects the anchor to be placed after the query parameters. This means that the `lid` value must be appended before the anchor for proper tracking, and since Braze reads the URL from left to right, the question mark (?) and `lid` should come before the anchor.
+
+| Link in Email Body                                                                        | Link with Aliasing                                                                                           |
+|-------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|
+| <a href="https://www.braze.com/promotions#special-offer">Check out our special offer!</a> | <a href="https://www.braze.com/promotions#special-offer?lid={{link_alias}}">Check out our special offer!</a> |
```
Comment (rachel-feinberg) [inline _docs/_user_guide/message_building_by_channel/email/templates/link_aliasing.md:125]: The links are displaying in the deployment!

**Review by rachel-feinberg — APPROVED** [review]
The final two links are parsing and are clickable, so the HTML isn't displaying

File: _docs/_user_guide/message_building_by_channel/email/templates/link_aliasing.md (hunk 1 above)
Comment (lydia-xie) [reply _docs/_user_guide/message_building_by_channel/email/templates/link_aliasing.md:125]: Oops, gonna wrap them in ` - thanks for flagging!

Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/templates/link_aliasing.md:125]:
```suggestion
| {%raw%}`<a href="https://www.braze.com/promotions#special-offer">Check out our special offer!</a>`{%endraw%}  | {%raw%}`<a href="https://www.braze.com/promotions#special-offer?lid={{link_alias}}">Check out our special offer!</a>` {%endraw%} |
```

**Comment by lydia-xie** [comment]
@cla-bot check

**Pull request merged**